organizations/serializers.py

```diff
@@ -10,9 +13,32 @@
 # pylint: disable=too-few-public-methods
 class OrganizationSerializer(serializers.ModelSerializer):
     """ Serializes the Organization object."""
+    logo_url = serializers.CharField(write_only=True, required=False)
+
     class Meta(object):  # pylint: disable=missing-docstring
         model = models.Organization
-        fields = '__all__'
+        fields = ('id', 'created', 'modified', 'name', 'short_name', 'description', 'logo',
+                  'active', 'logo_url',)
+
+    def update_logo(self, obj, logo_url):  # pylint: disable=missing-docstring
+        logo = requests.get(logo_url)
+        obj.logo.save(logo_url.split('/')[-1], ContentFile(logo.content))
+
+    def create(self, validated_data):
+        logo_url = validated_data.pop('logo_url', None)
+        obj = models.Organization.objects.create(**validated_data)
+        if logo_url:
+            self.update_logo(obj, logo_url)
+        return obj
+
+    def update(self, instance, validated_data):
+        logo_url = validated_data.pop('logo_url', None)
+        for key, value in validated_data.items():
+            setattr(instance, key, value)
+        instance.save()
```

Comment:
This can be replaced with a call to `super().update()` maybe? Same for `create` above?